src/stores/business.ts

```diff
@@ -123,6 +123,13 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
       `${apiURL}/businesses/${identifier}/addresses`, { params, dedupe: 'defer' })
       .then(({ data, error }) => {
         if (error.value || !data.value) {
+          // special case for missing business addresses
+          if (error.value?.statusCode === StatusCodes.NOT_FOUND &&
+            error.value?.data?.message?.includes('address not found')
+          ) {
+            return { businessOffice: null }
+          }
+
```

Comment:
This is essentially the same logic as we had in Filings UI, ie:

![image](https://github.com/user-attachments/assets/f4b19db2-c1ae-455c-8f81-f83263677368)

(`error.response` is now `error.value` and there is no `error.response.config`.)